specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -206,6 +208,48 @@ This can help to prevent cascading failures and ensure that issues are isolated
 There are more settings that are configurable by the user, that can be explored in the
 [config](/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py).
 
+### Capacity Estimation and Constraints
+
+The load to the Control Service APIs from the Meta Jobs plugin mostly depends on two factors:
+
+* Number of concurrent Data Job Executions (by default limited to 15, but it is
+  [configurable](/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py))
+* Data Job Executions duration
+
```

Comment:
It also depends on configuration values of META_JOBS_TIME_BETWEEN_STATUS_CHECK_SECONDS 

https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py#L76